app-text/dblatex/dblatex-0.3.10.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Transform DocBook using TeX macros"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
+HOMEPAGE="http://dblatex.sourceforge.net/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="-inkscape"
+
+RDEPEND="
+	app-text/texlive
+	dev-texlive/texlive-latexextra
+	dev-texlive/texlive-latexrecommended
+	|| ( dev-texlive/texlive-mathscience dev-texlive/texlive-mathextra )
+	dev-texlive/texlive-pictures
+	dev-texlive/texlive-xetex
+	dev-libs/libxslt
+	app-text/docbook-xml-dtd:4.5
+	gnome-base/librsvg
+	inkscape? ( media-gfx/inkscape )
+"
+DEPEND="${RDEPEND}"
+
+python_prepare_all() {
+	distutils-r1_python_prepare_all
+	use inkscape || eapply "${FILESDIR}/${P}-no-inkscape-dependency.patch"
+	eapply "${FILESDIR}/${PN}-path-logging.patch"
+	eapply "${FILESDIR}/${PN}-setup.patch"
+}
+
+python_install_all() {
+	python_doscript "${S}"/scripts/dblatex
+	python_optimize
```

Comment:
I tried moving this too. The setup.py is a bit hacky, which would require quite some work, possibly pushing it upstream. For now let's just let this be.